triggerbot/tree_watcher.py

```diff
@@ -145,12 +148,12 @@ def requested_trigger(self, branch, rev, builder):
 
     def add_rev(self, branch, rev, comments, user):
 
-        req_count, should_retry = self.triggers_from_msg(comments)
+        req_count, req_talos_count, should_retry = self.triggers_from_msg(comments)
 
         # Only trigger based on a request or a failure, not both.
-        if req_count:
+        if req_count or req_talos_count:
             self.log.info('Added %d triggers for %s' % (req_count, rev))
-            self.revmap[rev]['requested_trigger'] = req_count
+            self.revmap[rev]['requested_trigger'] = (req_count, req_talos_count)
         elif should_retry:
```

Comment:
It's sort of an edge case, but perfectly valid for someone to request talos retriggers and also expect their unittests to be retried on failure. This condition could be changed to ```if should_retry and not req_count:``` to accommodate that.